vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -65,12 +89,24 @@ void AppState::RemoveListeners(double /*count*/) noexcept {
 }
 
 ReactNativeSpecs::AppStateSpec_Constants AppState::GetConstants() noexcept {
-  return {m_active ? "active" : "background"};
+  return {m_enteredBackground ? "background" : m_deactived ? "inactive" : "active"};
+}
+
+void AppState::SetDeactived(bool deactived) noexcept {
```

Comment:
typo here and others: should be SetDeactivated, deactivated, m_deactivated

Reply:
I don't know how I missed that! Great catch!